lib/portlayer/exec/config.go

```diff
@@ -32,25 +31,13 @@ type Configuration struct {
 	DebugLevel int `vic:"0.1" scope:"read-only" key:"init/common/debug"`
 
 	// Port Layer - exec
-	// Default containerVM capacity
-	ContainerVMSize config.Resources `vic:"0.1" scope:"read-only" recurse:"depth=0"`
+	config.Container `vic:"0.1" scope:"read-only" key:"container"`
 
-	// Permitted datastore URLs for container storage for this virtual container host
-	ContainerStores []url.URL `vic:"0.1" scope:"read-only" recurse:"depth=0"`
-
-	// Resource pools under which all containers will be created
-	ComputeResources []types.ManagedObjectReference `vic:"0.1" scope:"read-only"`
 	// Resource pool is the working version of the compute resource config
 	ResourcePool *object.ResourcePool
 	// Parent resource will be a VirtualApp on VC
 	VirtualApp *object.VirtualApp
 
-	// Path of the ISO to use for bootstrapping containers
-	BootstrapImagePath string `vic:"0.1" scope:"read-only" key:"bootstrap_image_path"`
-
-	// Allow custom naming convention for containerVMs
-	ContainerNameConvention string
-
 	// FIXME: temporary work around for injecting network path of debug nic
```

Comment:
May as well remove Networks and DebugNetwork as well. I've submitted a PR to your branch for that.